client/components/Layout/FormEditScholarship.js

```diff
@@ -11,6 +12,13 @@ const FormEditScholarship = () => {
 		formState: { errors, defaultValues },
 		reset,
 		setValue,
+		setPage,
```

Comment:
setPage, sendData, error, gap is not in useForm

Reply:
มันไม่ได้อยู่อยุแล้วอ่ออออ

Reply:
https://react-hook-form.com/api/useform/

Reply:
setPage, sendData, error, gap ในหน้า register อันนั้นได้มาจาก props อีกทีอะ ไม่ได้มาจาก useForm()

Reply:
อ่อออ หมายถึงให้ลบออกอ่อ

Reply:
ใช่ๆ :+1: